MiniSim/Model/Device.swift

```diff
@@ -8,49 +8,49 @@
 struct Device: Hashable, Codable {
     var name: String
     var version: String?
-    var ID: String?
-    var booted: Bool = false
+    var identifier: String?
```

Comment:
How critical is changed name of this property? Do we rely on persisted value?

Reply:
Sorry for the delay, the devices are persisted in Raycast extension for Minisim. I will check if this breaks the extension.

Reply:
I think we can keep the `identifier` - Raycast extension is working properly